addons/mail/models/mail_template.py

```diff
@@ -178,6 +178,8 @@ def generate_email(self, res_ids, fields):
                 values = results[res_id]
                 if values.get('body_html'):
                     values['body'] = tools.html_sanitize(values['body_html'])
+                if 'scheduled_date' in fields:
```

Comment:
Why should we do this ? We already render all fields previously (for field in fields) .